Library/Formula/rtags.rb

```diff
@@ -0,0 +1,31 @@
+class Rtags < Formula
+  desc "ctags-like source code cross-referencer with a clang frontend"
+  homepage "https://github.com/Andersbakken/rtags"
+  url "https://github.com/Andersbakken/rtags.git",
+      :tag => "v2.0",
+      :revision => "ba85598841648490e64246be802fc2dcdd45bc3c"
+
+  head "https://github.com/Andersbakken/rtags.git"
+
+  depends_on "cmake" => :build
+  depends_on "llvm" => "with-clang"
+  depends_on "openssl"
+
+  def install
+    # Homebrew llvm libc++.dylib doesn't correctly reexport libc++abi
+    ENV.append("LDFLAGS", "-lc++abi")
+
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    # not using shell_output because on HEAD the exit code will be 0, but on
+    # stable it will be 1.
+    cmd = "#{bin}/rc --help"
+    assert_match /rc options/, `#{cmd}`
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!